src/python/test/test_dxclient.py

```diff
@@ -1192,6 +1192,21 @@ def test_rename(self):
 
 
 class TestDXClientUploadDownload(DXTestCase):
+    def test_dx_download_recursive_overwrite(self):
+        if os.path.exists("foodir/"):
+            run("rm -rf foodir")
+        wd = "foodir"
```

Comment:
Please move `wd = "foodir"` to the top and use it in the subsequent strings.  Thanks!